subsystems/drivesubsystem.py

```diff
@@ -615,7 +615,7 @@ def arcadeDriveWithSpeeds(
                 chassisSpeeds.vx,
                 chassisSpeeds.vy,
                 chassisSpeeds.omega,
-                self.odometry.getPose().rotation(),
+                self.getRotation(),
```

Comment:
Can you elaborate on the thought originally on using odometry vs gyro?

Reply:
in competitions and in simulation, the odometry's position's rotation drifts compared to the gyro and the actual robot. The gyro is an absolute measurement that does not worry about cumulative error as much.